app/move/controllers/update/court.js

```diff
@@ -0,0 +1,24 @@
+const { get } = require('lodash')
+
+const AssessmentController = require('./assessment')
+
+class UpdateCourtController extends AssessmentController {
+  // TODO: replace this with a centralised more form-wizard-based protection
+  configure(req, res, next) {
+    const isCourt =
+      get(res, 'locals.move.to_location.location_type') === 'court'
+
+    if (!isCourt) {
+      const error = new Error(
+        'Forbidden. Updating court information is not allowed for this move'
+      )
+      error.statusCode = 403
```

Comment:
I wonder if this should `404` instead. We're saying that this information doesn't exist for this move type so maybe a `404` is a better error code?

Or we do what the form wizard does and just redirect past it.

Reply:
Now `404`s